benchmarks/bench_cutlass_fused_moe.py

```diff
@@ -181,8 +172,9 @@ def bench_cutlass_fused_moe(
                 quant_scales=quant_scales,
                 input_sf=input_sf,
                 output=flash_output,
+                tune_max_num_tokens=16384,
             )
-    ms = do_bench(
+    ms = do_bench_cudagraph(
```

Comment:
We can use https://github.com/flashinfer-ai/flashinfer/blob/43e08e9b0fa58b4d51998ce20ca439771c8ac200/flashinfer/testing/utils.py#L610 contributed by @bkryu et al.

Reply:
Done.